cmd/minikube/cmd/service_list.go

```diff
@@ -66,14 +66,15 @@ var serviceListCmd = &cobra.Command{
 			if len(serviceURL.URLs) == 0 {
 				data = append(data, []string{serviceURL.Namespace, serviceURL.Name, "No node port"})
 			} else {
+				servicePortNames := strings.Join(serviceURL.PortNames, "\n")
 				serviceURLs := strings.Join(serviceURL.URLs, "\n")
 
 				// if we are running Docker on OSX we empty the internal service URLs
 				if runtime.GOOS == "darwin" && cfg.Driver == oci.Docker {
 					serviceURLs = ""
 				}
 
-				data = append(data, []string{serviceURL.Namespace, serviceURL.Name, "", serviceURLs})
+				data = append(data, []string{serviceURL.Namespace, serviceURL.Name, servicePortNames, serviceURLs})
```

Comment:
What we print for `minikube service list` must be consistent with what we print for `minikube service <service-name>`, so if we are printing `name || port` there, the same should be done here, but consider my recommendation to do it like `name/port`

Reply:
Both `minikube service list` and `minikube service <service-name>` eventually call `printURLsForService` which is where I am making these changes. So the results should be consistent. Maybe the variable name needs to change?